ui/src/components/yaml/index.tsx

```diff
@@ -377,35 +380,45 @@
                   />
                 </Space>
               </div>
-              <div className={styles.yaml_content_diagnosisContent}>
-                {interpretStatus === 'loading' ||
-                (interpretStatus === 'streaming' && !interpret) ? (
-                  <div className={styles.yaml_content_diagnosisLoading}>
-                    <Spin />
-                    <p>{t('EventAggregator.DiagnosisInProgress')}</p>
-                  </div>
-                ) : interpretStatus === 'streaming' ? (
-                  <div>
-                    <Markdown>{interpret}</Markdown>
-                    <div className={styles.yaml_content_streamingIndicator}>
-                      <span className={styles.dot}></span>
-                      <span className={styles.dot}></span>
-                      <span className={styles.dot}></span>
+              <div className={styles.yaml_content_diagnosisBody}>
+                <div
+                  className={styles.yaml_content_diagnosisContent}
+                  ref={contentRef}
+                >
+                  {interpretStatus === 'loading' ||
+                  (interpretStatus === 'streaming' && !interpret) ? (
```

Comment:
The use of a fragment here is correct, but verify that any parent components or styles do not expect a single child element.